tests/unit-tests/test_dataloading.py

```diff
@@ -532,9 +544,62 @@ def test_GSgnnData_edge_feat():
                 edge_feat_list = {}
             assert len(edge_feat_list) == 0
 
+        # Test 5: layer 0 has edge feature, and layer 1 not, i.e., empty dict
+        target_idx = {'n0': [0]}
+        dataloader = GSgnnNodeDataLoader(gdata, target_idx, [100, 100], 10,
+                                        label_field='label',
+                                        train_task=False)
+        efeat_fields = {
+            ('n0', 'r0', 'n1'): ['feat'],
+            ('n0', 'r1', 'n1'): ['feat']
+        }
+        for _, _, blocks in dataloader:
+            edge_feat_list = gdata.get_blocks_edge_feats(blocks, efeat_fields)
+            assert len(edge_feat_list) == 2
+            assert edge_feat_list[1] == {}
+
     # after test pass, destroy all process group
     th.distributed.destroy_process_group()
 
+def test_GSgnnData_edge_feat2():
+    """ Test GSgnnData built-in functions.
+    
```

Comment:
Can you be more specific about the difference between test_GSgnnData_edge_feat and test_GSgnnData_edge_feat2?

Reply:
add more descriptions.